examples/code/home/configuration/engine/config_template.yaml

```diff
@@ -1,56 +1,14 @@
-telemetry:
-  environment: ${env:TELEMETRY_ENVIRONMENT}
-  version: ${env:VERSION}
-  logging:
-    level: ${env:TELEMETRY_LOGGING_LEVEL} # debug, info, warn, error
-    encoding: ${env:TELEMETRY_LOGGING_ENCODING}
```

Comment:
Changes in this file:
- Sorted top-level keys alphabetically: `api, auth, ...`
- Added missing required sections: `security, storage`
- Fixed small typos, etc. that were caught by schema validation